lib/date_logic.rb

```diff
@@ -0,0 +1,33 @@
+module DateLogic
```

Comment:
This is one of the most appropriate ways to use a module and I'm very very very happy with this.